### DISCUSSION TITLE
skoala/registry/managed/registry-lcm/create-registry/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1545] [Announcements]
# skoala/registry/managed/registry-lcm/create-registry/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/registry/managed/registry-lcm/create-registry/

**Comment by liyuerich** [comment]
1核2G 指 CPU 的请求值和限制值分别为 2 核，内存的请求值和限制值分别为 2 G，以此类推。

此处 “CPU 的请求值和限制值分别为 2 核”， 应该为 1核 吧？

**Comment by liyuerich** [comment]
如果创建注册中心失败，能举例怎么定位问题吗？

**Comment by liyuerich** [comment]
没有看到注册中心概览的介绍？

**Reply by Michelle951** [reply]
查看概览页面：https://docs.daocloud.io/skoala/registry/managed/registry-lcm/check-registry/
